src/styles/blocks/header.scss

```diff
@@ -0,0 +1,20 @@
+.header {
+  grid-area: 1 / 1 / 2 / -1;
+
+  display: grid;
+  grid-template-columns: auto min-content min-content;
+  grid-template-rows: min-content auto;
+  gap: 30px;
+  height: 480px;
+  padding: 29px 20px 88px;
+  background-image: url("../images/image3.jpg");
+
+  &__h1 {
```

Comment:
rename to `title`

Reply:
![image](https://user-images.githubusercontent.com/60798246/182231150-5755dc6f-3e36-4643-aaa2-42ee3e11557e.png)